packages/graphback-runtime/src/service/GraphbackCRUDService.ts

```diff
@@ -18,83 +21,86 @@ export interface ResultList<T = any> {
  */
 //tslint:disable-next-line: no-any
 export interface GraphbackCRUDService<Type = any, GraphbackContext = any> {
+  setModelConfig(modelType: GraphQLObjectType, pubSubConfig: GraphbackPubSubModel): void
```

Comment:
Not entirely sure if it is a good idea to have these setters in the CRUD service interface.